grammars/tree-sitter-python.cson

```diff
@@ -66,44 +66,220 @@ scopes:
   'none': 'constant.language'
   'true': 'constant.language'
   'false': 'constant.language'
-  'integer': 'constant.language'
-  'float': 'constant.language'
+  'integer': 'constant.numeric'
+  'float': 'constant.numeric'
 
   'type > identifier': 'support.storage.type'
 
   '"class"': 'storage.type.class'
   '"def"': 'storage.type.function'
   '"lambda"': 'storage.type.function'
+  '"nonlocal"': 'storage.modifier.nonlocal.python'
+  '"global"': 'storage.modifier.global.python'
 
-  '"if"': 'keyword.control'
-  '"else"': 'keyword.control'
-  '"elif"': 'keyword.control'
-  '"while"': 'keyword.control'
-  '"for"': 'keyword.control'
-  '"return"': 'keyword.control'
-  '"break"': 'keyword.control'
-  '"continue"': 'keyword.control'
-  '"raise"': 'keyword.control'
-  '"yield"': 'keyword.control'
-  '"await"': 'keyword.control'
-  '"async"': 'keyword.control'
-  '"try"': 'keyword.control'
-  '"except"': 'keyword.control'
-  '"with"': 'keyword.control'
-  '"as"': 'keyword.control'
-  '"finally"': 'keyword.control'
-  '"import"': 'keyword.control'
-  '"from"': 'keyword.control'
-  '"print"': 'keyword.control'
-  '"assert"': 'keyword.control'
+  '"import"': 'keyword.control.import.from.python'
+
+  '"elif"': 'keyword.control.conditional.python'
+  '"else"': 'keyword.control.conditional.python'
+  '"if"': 'keyword.control.conditional.python'
+
+  '"except"': 'keyword.control.exception.python'
+  '"finally"': 'keyword.control.exception.python'
+  '"try"': 'keyword.control.exception.python'
+
+  '"for"': 'keyword.control.repeat.python'
+  '"while"': 'keyword.control.repeat.python'
+
+  '"return"': 'keyword.control.statement.python'
+  '"break"': 'keyword.control.statement.python'
+  '"continue"': 'keyword.control.statement.python'
+  '"raise"': 'keyword.control.statement.python'
+  '"yield"': 'keyword.control.statement.python'
+  '"await"': 'keyword.control.statement.python'
+  '"with"': 'keyword.control.statement.python'
+
+  '"as"': 'keyword.other.python'
+  '"assert"': 'keyword.other.python'
+  '"del"': 'keyword.other.python'
+  '"async"': 'keyword.other.python'
+  '"from"': 'keyword.other.python'
+
+  # Those are handled as "built-in functions" below
+  # '"exec"': 'keyword.deprecated.python'
+  # '"print"': 'keyword.deprecated.python'
```

Comment:
I think we should leave these highlighted, as this highlighting will still apply to Python 2 source files. I believe there should be no problem; Tree-sitter should detect when `print` and `exec` are being used as functions and parse them as simple `identifiers`, so they won't get highlighted as keywords in that case.

Reply:
You're right, I'll restore this.